src/utils/autoBotEngine.js

```diff
@@ -121,7 +121,11 @@ const getFallbackOptionText = (personality, id) => {
   }
   return `${
     personalityObj.fallback.text
-  }${DEFAULT_PERSONALITY_FALLBACK_SEPARATOR}${getFormattedOptionText(step)}`;
+  }${DEFAULT_PERSONALITY_FALLBACK_SEPARATOR}${getFormattedOptionText(
+    step,
+  )}${DEFAULT_PERSONALITY_FALLBACK_SEPARATOR}You can reply with "${step.options.join(
```

Comment:
Can leave this for now, but maybe translate `You can reply with` and add a colon at the end`:`.